incubator/redis-cache/values.yaml

```diff
@@ -0,0 +1,24 @@
+# Default values for redis-cache.
```

Comment:
Ooops i misunderstood and re-categorized values based on kubernetes objects like service, pdb, statefulsets etcd which had to be done too.  Now it is grouped based on components.  Please take a look.  Also, let me know if you want the commits them to squashed.